rules/library.bzl

```diff
@@ -412,6 +444,21 @@ def _find_imported_framework_name(outputs):
         return fw_name
     return None
 
+def _find_imported_xcframework_name(outputs):
+    """
+    Returns the name of the .xcframework that includes the given outputs.
+
+    Note: for a static library there is no .framework.
+    Instead, the .xcframework is the name of the static library.
+    """
+    for output in outputs:
+        if not ".xcframework" in output:
+            continue
+        prefix = output.split(".xcframework/")[0]
+        fw_name = prefix.split("/")[-1]
+        return fw_name
+    return None
```

Comment:
Without this function, for static libraries there is no `.framework` so the existing `_find_imported_framework_name` always returned `None` which leads to the wrong name for the "framework" in the VFS

Reply:
At one point someone tried to make a nested xcframework - that should also probably be an error 👁️ . There isn't a spec for `.xcframework` but it seemed wrong. I would recommend to make an explicit error if they try to feed it code that looks like that: e.g. if you split `output` - it has many parts